src/app/shared/pipes/text-transform.pipe.ts

```diff
@@ -0,0 +1,20 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'textTransform'
```

Comment:
Stillman naming?
You transform text into what?
The naming should reflect the purpose